app/views/account/deliveryTrackingSuccess.scala.html

```diff
@@ -0,0 +1,36 @@
+@import model.FulfilmentLookup
+@import org.joda.time.LocalDate
+@import views.support.Dates._
+@(
+    fulfilmentLookup: FulfilmentLookup,
+    date: LocalDate
+)
+
+@main("Delivery Status | The Guardian", bodyClasses=List("is-wide")) {
+    <main class="page-container gs-container gs-container--slim">
+        @fragments.page.header("Delivery Status", None, List("l-padded"))
+        <section class="section-slice section-slice--bleed section-slice--limited">
+            @if(fulfilmentLookup.subscriptionInFile) {
+                <p>
+                    Your issue is out for delivery.
```

Comment:
I'd prefer something like 
'Your issue was allocated for delivery on datedatedate.  If you have still not received your issue after the expected date, please contact....etcetc'